contrib/internet-urllib3.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+import urllib3
+import certifi
```

Comment:
Yes, certifi is only imported so that this code will run self contained from the command line.  In actual use, the function can re-use the path to `cacert.pem` which is already provided in the offlineimap config file.